css/styles.css

```diff
@@ -39,3 +39,7 @@
 #password_policy .margin-add-top {
 	margin-top: 24px;
 }
+
+#password_policy #password_hint {
```

Comment:
`#password_hint` should be enough as ids are unique anyway

Reply:
Let us stick with the syntax in the whole CSS file #password_policy ....{}